config/templates/cloud-config-worker

```diff
@@ -144,27 +144,32 @@ coreos:
     {{ if .Experimental.NodeDrainer.Enabled }}
     - name: kube-node-drainer.service
       enable: true
+      command: start
+      runtime: true
       content: |
         [Unit]
         Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
-        Wants=kubelet.service docker.service
+        After=multi-user.target
 
         [Service]
         Type=oneshot
         RemainAfterExit=true
-        ExecStartPre=/usr/bin/systemctl is-active kubelet.service
-        ExecStartPre=/usr/bin/systemctl is-active docker.service
         ExecStart=/bin/true
-        ExecStop=/bin/sh -c '/usr/bin/docker run --rm -v /etc/kubernetes:/etc/kubernetes {{.HyperkubeImageRepo}}:{{.K8sVer}} \
-          /hyperkube kubectl \
+        TimeoutStopSec=30s
+        ExecStop=/bin/sh -c '/usr/bin/rkt run \
+        --volume=kube,kind=host,source=/etc/kubernetes,readOnly=true \
+        --mount=volume=kube,target=/etc/kubernetes \
+        --net=host \
+        quay.io/coreos/hyperkube:v1.4.5_coreos.0 \
```

Comment:
Would it be ok to keep using `{{.HyperkubeImageRepo}}:{{.K8sVer}}` instead of `quay.io/coreos/hyperkube:v1.4.5_coreos.0` to avoid hard-coding for future ease in maintenance/more customizability?